### PR TITLE
v48.7.0 Changelog

### PR DESCRIPTION
**Pull request by jag-sfdc**
### What does this PR do?
Update v48.7.0 Changelog

### CODE REVIEW

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,22 @@
+# 48.7.0 - March 19, 2020
+
+## Fixed
+#### salesforcedx-vscode-core
+
+- Update to latest version of @salesforce/core ([PR #2077](https://github.com/forcedotcom/salesforcedx-vscode/pull/2077))
+
+#### docs
+
+- Add linting topic to LWC parent. ([PR #2060](https://github.com/forcedotcom/salesforcedx-vscode/pull/2060))
```
Comment (AnanyaJha) [inline packages/salesforcedx-vscode/CHANGELOG.md]: Can we remove the period at the end of the sentence?

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,22 @@
...
+
+- Minor tweaks after release ([PR #2072](https://github.com/forcedotcom/salesforcedx-vscode/pull/2072))
+
+#### salesforcedx-vscode
+
+- Minor tweaks after release ([PR #2072](https://github.com/forcedotcom/salesforcedx-vscode/pull/2072))
```
Comment (AnanyaJha) [inline packages/salesforcedx-vscode/CHANGELOG.md]: I think we're missing an entry for the interactive debugger fix that went in

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,22 @@
+# 48.7.0 - March 19, 2020
+
+## Fixed
+#### salesforcedx-vscode-core
+
+- Update to latest version of @salesforce/core ([PR #2077](https://github.com/forcedotcom/salesforcedx-vscode/pull/2077))
```
Comment (lcampos) [inline packages/salesforcedx-vscode/CHANGELOG.md]: This change does not need to be documented, let's remove it.

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,22 @@
...
+
+#### docs
+
+- Add linting topic to LWC parent. ([PR #2060](https://github.com/forcedotcom/salesforcedx-vscode/pull/2060))
+
+- Minor tweaks after release ([PR #2072](https://github.com/forcedotcom/salesforcedx-vscode/pull/2072))
```
Comment (lcampos) [inline packages/salesforcedx-vscode/CHANGELOG.md]: The lines for change from PR 2072 should be removed from the changelog.

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 5]
```diff
@@ -1,3 +1,14 @@
+# 48.7.0 - March 19, 2020
+
+## Fixed
+#### salesforcedx-vscode-core
+
+#### docs
+
+- Add linting topic to LWC parent ([PR #2060](https://github.com/forcedotcom/salesforcedx-vscode/pull/2060))
+
+## Added
```
Comment (lcampos) [inline packages/salesforcedx-vscode/CHANGELOG.md]: You can remove this `## Added`.

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 6]
```diff
@@ -1,3 +1,14 @@
+# 48.7.0 - March 19, 2020
+
+## Fixed
+#### salesforcedx-vscode-core
+
```
Comment (lcampos) [inline packages/salesforcedx-vscode/CHANGELOG.md]: We're missing the fix from PR #2083.

Comment (lcampos) [inline packages/salesforcedx-vscode/CHANGELOG.md]:
```suggestion
#### salesforcedx-vscode-core

- Fix empty error message when running Apex Interactive Debugger with incorrect user permissions ([PR #2083](https://github.com/forcedotcom/salesforcedx-vscode/pull/2083))
```

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 7]
```diff
@@ -1,3 +1,14 @@
...
+#### docs
+
+- Add linting topic to LWC parent ([PR #2060](https://github.com/forcedotcom/salesforcedx-vscode/pull/2060))
+
+- Port the interactive debugger changes to the release branch ([PR #2083](https://github.com/forcedotcom/salesforcedx-vscode/pull/2083))
+
```
Comment (lcampos) [inline packages/salesforcedx-vscode/CHANGELOG.md]: This line needs to be moved to the code section under ## Fixed

**Pull request merged**